doc/use_cases.md

```diff
@@ -46,22 +52,38 @@ For each jugement object returned by:
 
     curl http://ctiahost/ctia/ip/192.168.1.1/indicators
 
-Extract the IDs from the `related_campaigns.campaign__id' fields and
+Extract the IDs from the 'related_campaigns.campaign_id' fields and
 
     curl http://ctiahost/ctia/campaigns/ID
 
-### 2.3 As an incident responder, I would like to know what suggest COAs are asociated with an IP
+### 2.3 As an incident responder, I would like to know what suggested COAs are asociated with an IP
 
-For each jugement object returned by:
+For each indicator object returned by:
 
     curl http://ctiahost/ctia/ip/192.168.1.1/indicators
 
-Extract the IDs from the `suggested_COA.COA__id' fields and
+Extract the IDs from the 'suggested_COA.COA__id' fields and
 
     curl http://ctiahost/ctia/coa/ID
 
 ### 2.4 As an incident responder, I would like to record an incident and it's sightings
 
+Determine the ID of the incident's indicator(s).  For example, if you know the title:
+
+	curl http://ctiahost/ctia/indicator/title/document-direct-ip-traffic
+
+Or search for the indicator:
+
+    curl http://ctiahost/ctia/indicator/_search?title=ip # URL TBD
```

Comment:
I have a simple question as it is related to my current WIP on searching indicator.

I know that we should use `GET` but, if we want to remove the size of the request limit and also have a better swagger integration and typing coercion. We might prefer to provide complex search expression via `POST` instead.

Imagine a complex request containing many different dimension all with a large cardinality.
Is this something we want to support?